mkl_fft/interfaces/_scipy_fft.py

```diff
@@ -203,6 +203,17 @@ def _init_nd_shape_and_axes(x, shape, axes, invreal=False):
     return tuple(shape), list(axes)
 
 
+def _use_input_as_out(x, overwrite_x):
+    """Check if the input can be used as output."""
+    if overwrite_x and np.issubdtype(x.dtype, np.complexfloating):
+        # pass input as out to overwrite it
+        out = x
+    else:
+        out = None
+
+    return out
```

Comment:
```suggestion
    if overwrite_x and np.issubdtype(x.dtype, np.complexfloating):
        # pass input as out to overwrite it
        return x
    else:
        return None
```
could be simplified like this